tests/services/scanner/scanner_dao_test.py

```diff
@@ -88,7 +108,8 @@ def tearDown(self):
 
     def test_save_violations(self):
         """Test violations can be saved."""
-        saved_violations = populate_db().list()
+        access, _ = populate_db()
```

Comment:
`access` should be `violation_access` to match what is in the application code.  Please change this everywhere.

Reply:
Done.